### PR TITLE
add COW token

### LINKED ISSUE #803: Add COW token to tokenlist
**Description**
Add COW token to token list.

### PR DESCRIPTION
**Pull request by 0xVenky**
# Summary

Fixes #803 

  # To Test

Check if you are able to see COW token in the token list in Mainnet and Gnosis chain.
image

### CODE REVIEW

File: src/constants/index.tsx [hunk 1]
```diff
@@ -256,7 +256,7 @@ export const BLOCKED_PRICE_IMPACT_NON_EXPERT: Percent = new Percent(JSBI.BigInt(
 // used to ensure the user doesn't send so much ETH so they end up with <.01
 export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)) // .01 ETH
 
-export const DEFAULT_TOKEN_LIST = 'ipfs://QmSbyVo6Kz5BuqyAHYcN7UkeCk5cALFp6QmPUN6NtPpDWL'
+export const DEFAULT_TOKEN_LIST = 'ipfs://QmbZ2F2miMNxfNjSpigmAMUpaTJSFdoPD2cBCJnB5HHFW1'
```
Comment (adamazad) [inline src/constants/index.tsx:259]: A slight typo here:

name: CoW Protocol Token

image

Ref in contract

image
https://etherscan.io/address/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB#readContract

**Review by adamazad — CHANGES_REQUESTED** [review]

**Comment by MilanVojnovic95** [comment]
Tested and looks ok.
Present on the list of tokens on the Gnosis chain and Mainnet.

File: src/constants/index.tsx (hunk 1 above)
Comment (0xVenky) [reply src/constants/index.tsx]: Fixed now with updated token name.

File: src/constants/index.tsx [hunk 2]
```diff
@@ -256,7 +256,7 @@ export const BLOCKED_PRICE_IMPACT_NON_EXPERT: Percent = new Percent(JSBI.BigInt(
 // used to ensure the user doesn't send so much ETH so they end up with <.01
 export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)) // .01 ETH
 
-export const DEFAULT_TOKEN_LIST = 'ipfs://QmSbyVo6Kz5BuqyAHYcN7UkeCk5cALFp6QmPUN6NtPpDWL'
+export const DEFAULT_TOKEN_LIST = 'ipfs://QmUuUz4ZFwK2KVukPzBr8AmuotArkBZmq1RaNp9E5KTJvz'
```
Comment (adamazad) [inline src/constants/index.tsx:259]:
```suggestion
export const DEFAULT_TOKEN_LIST = 'ipfs://QmUWnK6AFHZ3S1hR7Up1h3Ntax3fP1ZyiTptDNG2cWLTeK'
```

**Pull request merged**